draft-ietf-tls-esni.md

```diff
@@ -699,8 +699,10 @@ MAY offer to resume sessions established without ECH.
 ## Client-Facing Server
 
 Upon receiving an "encrypted_client_hello" extension, the client-facing server
-MUST check that it is able to negotiate TLS 1.3 or greater. If not, it MUST
-abort the connection with a "handshake_failure" alert.
+determines if it can decrypt it, prior to negotiating any other TLS parameters.
```

Comment:
s/can decrypt it/can accept ECH/, since "accept ECH" is a well-defined term.

Reply:
(Resolved by committing @chris-wood's suggestion.)